template/luos_hal.c

```diff
@@ -1,6 +1,6 @@
 /******************************************************************************
  * @file luosHAL
- * @brief Luos Hardware Abstration Layer. Describe Low layer fonction 
```

Comment:
This commit is ok but you should add some more comment describing all the operation users have to do.
Also there is some inconsistency on comment format, you could use this commit to make it right.